docs/deployment-process/variables/system-variables.md

```diff
@@ -26,7 +26,7 @@ Deployment-level variables are drawn from the project and release being deployed
 |**`Octopus.Deployment.Id`** <br/>The ID of the deployment | *deployments-123*|
 |**`Octopus.Deployment.Name`** <br/>The name of the deployment | *Deploy to Production*|
 |**`Octopus.Deployment.PreviousSuccessful.Id`** <br/>The ID of the previous successful deployment of this project in the target environment | *deployments-122*|
-|**`Octopus.Deployment.Machines`** <br/>Ids of machines being targeted by the deployment. | machines-123,machines-124|
+!partial <deploymentMachineVariables>
```

Comment:
I think the current approach is to not use partials anymore, rather just note the version number. Check with @leecarlon

Reply:
Oh, OK. I was not aware of that

Reply:
Rob's correct, but a bit ahead of the curve. We won't need to use partials for old versions when the new versioned version of the docs go live (ETA next week, with details to follow), and in fact, at that point, using partials will only really be useful for pre-release content.

Reply:
OK, I'll revert it to just mention the version it was introduced in in the descripition